database/migrations/2025_01_08_124448_create_board.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('board', function (Blueprint $table) {
```

Comment:
```suggestion
        Schema::create('boards', function (Blueprint $table) {
```
اسم تمام جداول باید بصورت جمع باشه چون مجموعه ایی از برد هارو در یک جدول نگهداری میشه
سایر جاها فراموش نشه